### PR TITLE
Developmment

### PR DESCRIPTION
**Pull request by hunter4466**
Created about.html and index html
Added styles on about.css and index.css
Added functionality with about.js, index.js, and menu.js with javascript.
Fixed linters errors.
Finished project.

### CODE REVIEW

**Review by henatan99 — CHANGES_REQUESTED** [review]
# Status: Invalid :warning: 

You have a failing linters check. Please make sure all GitHub linter checks have passed before requesting code review.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <div id="menu_display" class="menu_display">
+        <img id="close_menu_btn" class="close_menu_btn" alt="close" src="./Assets/Images/Icons/close.png">
+        <ul class="md_links">
+            <li class="md-item"><a class="md-link-asside" href="./about.html">About</a></li>
+            <li class="md-item"><a class="md-link" href="#sec_2">Main Program</a></li>
+            <li class="md-item"><a class="md-link" href="#sec_3">Featured Speakers</a></li>
+        </ul>
+    </div>
+    <section id="sec_1" class="sec_1">
+        <h1 class="sec_1_greetings">"Hello! Chefs community"</h1>
```
Comment (Davidosky007) [inline index.html:41]: * This line of text (hello chefs community) is not  visible, please  adjust it 🙏 
Screenshot (123)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        <h3 class="sec_1_location">@Exposition park, Sheraton conference room and more</h3>
+    </section>
+    <section id="sec_2"  class="sec_2">
+        <h1 class="sec_2_title">Main Program</h1>
+        <div class="line_separator"></div>
+        <ul class="program_boxes">
+            <li class="pr_box">
+                <img class="pr_box_icon" alt="Lecture" src="./Assets/Images/Icons/lecture.png">
+                <h1 class="pr_box_title">Lecture</h1>
+                <p class="pr_box_parraph">Listen to chefs around the world about their techniques and the way they walkthrough kitchen issues.</p>
```
Comment (Davidosky007) [inline index.html]: * These are articles and should be wrapped in `article` tags, please make the correction in other places it reoccurs

File: Assets/Styles/index.css [hunk 3]
```diff
@@ -0,0 +1,1062 @@
...
+  display: none;
+}
+
+.mobile_navbar {
+  position: fixed;
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  width: 100vw;
+  height: 18vw;
+  top: 0.001vw;
```
Comment (Davidosky007) [inline Assets/Styles/index.css:163]: * I suggest you use VW unit carefully as it takes the whole available width of your viewport you should subtract padding and margins before using this unit . This is making your page overflow kindly adjust the viewport unit.
* ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Screenshot (125)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @hunter4466, 

Good job so far going beyond the required requirements 💪 !
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Please fix the overflowing images and contents causing your page to overflow check comments below for hints on how to fix it
* Pull request should have a descriptive title please add one 🙏 
* The link to your Video presentation is missing, please attach it to your PR 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html:41]: * Good job making the requested changes but i can see your middle section on mobile view has some issues 
* YOURS:
Screenshot (127)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ORIGINAL :
Screenshot (128)

File: Assets/Styles/index.css (hunk 3 above)
Comment (Davidosky007) [reply Assets/Styles/index.css:163]: * Well done fixing this 👍

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @hunter4466, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
* Please fix this one issue and submit it for review, you almost done 👍 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**